src/DIRAC/Core/Utilities/TimeUtilities.py

```diff
@@ -19,6 +19,7 @@
 if a give datetime is in the defined interval.
 
 """
+import calendar
 import time as nativetime
```

Comment:
Now there's no need to re-define `time`
```suggestion
import time
```